appimage_metadata/net.pioneerspacesim.Pioneer.desktop.cmakein

```diff
@@ -0,0 +1,10 @@
+[Desktop Entry]
+Version=1.0
+Name=Pioneer
+GenericName=Pioneer Space Simulator
+Comment=A game of lonely space adventure
+Exec=init_pioneer.sh
+Icon=net.pioneerspacesim.Pioneer
+Terminal=false
+Type=Application
+Categories=Game;AdventureGame;ActionGame;Simulation;
```

Comment:
You don't need a separate metadata file. Just use the existing `metadata/net.pioneerspacesim.Pioneer.desktop.cmakein` file, and just change how the Exec path is passed.

For instance:
```
Exec=@PIONEER_EXECUTABLE@
```

Then in CMake, just `set(PIONEER_EXECUTABLE ${CMAKE_INSTALL_FULL_BINDIR}/pioneer)` for the regular build, or `set(PIONEER_EXECUTABLE init_pioneer.sh)` for the AppImage build.